src/kvstore/raftex/RaftPart.cpp

```diff
@@ -775,7 +775,7 @@ void RaftPart::replicateLogs(folly::EventBase* eb,
             return resp.get_error_code() == cpp2::ErrorCode::SUCCEEDED
                     && !hosts[index]->isLearner();
         })
-        .then(executor_.get(), [self = shared_from_this(),
+        .via(executor_.get()).then([self = shared_from_this(),
```

Comment:
Should using `thenValue` to replace `then` ?

Reply:
Because we need to handle the exception in folly::Try,  so we should use "then" in this place.